_docs-sources/guides/production-framework/ingredients/ci-cd-pipeline/intro.md

```diff
@@ -0,0 +1,7 @@
+# CI / CD Pipeline
+
+One trend you may have noticed in this guide is that automation is preferred over manual processes. Most companies that
+succeed with the cloud invest heavily in setting up a Continuous Integration / Continuous Delivery pipeline (CI / CD
+pipeline) to automate their build, test, and deployment processes.
+
+![Gruntwork Pipelines](/img/guides/production-framework/gruntwork-pipelines.png)
```

Comment:
I think this section is missing a paragraph. You're showing me this image of the Pipelines and it's very vague. I think we could do better if we had a paragraph along the lines of: Because we understand the need to automate...we created the Gruntwork Pipelines product...then we show the diagram and link them to product docs and the guide of how to set them up for their own infra.